src/monero_key.c

```diff
@@ -215,8 +215,8 @@ int monero_apdu_display_address() {
     if (minor | major) {
         monero_get_subaddress(C, D, index);
     } else {
-        os_memmove(C, G_monero_vstate.A, 32);
-        os_memmove(D, G_monero_vstate.B, 32);
+        memcpy(C, G_monero_vstate.A, 32);
```

Comment:
Small remark, in cases like this how can we ensure that the value `C` computed does not overlap with `G_monero_vstate.A`

Reply:
`G_monero_vstate.A` is a 32-byte global array, while `C` and `D` are 32-byte local arrays. They cannot overlap. Or I miss something?